src/main/java/mate/academy/onlinebookstore/controller/AuthenticationController.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.onlinebookstore.controller;
+
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import mate.academy.onlinebookstore.dto.UserRegistrationRequestDto;
+import mate.academy.onlinebookstore.dto.UserResponseDto;
+import mate.academy.onlinebookstore.exception.RegistrationException;
+import mate.academy.onlinebookstore.service.UserService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/auth")
+public class AuthenticationController {
+    private final UserService userService;
+
+    @PostMapping("/registration")
+    public UserResponseDto register(@RequestBody @Valid UserRegistrationRequestDto requestDto)
+            throws RegistrationException {
```

Comment:
make exception unchecked
```suggestion
           {
```